client/src/shared/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,41 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+
+interface SearchBarProps {
+    onSearch: (query: string) => void;
+    height?: string;
+    width?: string;
+}
+
+function Searchbar({onSearch, height, width}: SearchBarProps) {
```

Comment:
- please set the default values
- change to `onSearchEvent`
```suggestion
function Searchbar({onSearchEvent, height, width}: SearchBarProps) {
```